### ISSUE TITLE
ComboBox with multiple selected values are not read aloud

### ISSUE THREAD

**Issue by mirajp** [OfficeDev/office-ui-fabric-react#10718]
### Environment Information

- **Package version(s)**: version 7 (latest at time of writing = 7.42.0)
- **Browser and OS versions**: Chrome 77.0.3865.90 and Edge 44.18362.329.0 (on Windows 10)

### Describe the issue:
Neither Narrator with Edge nor NVDA with Chrome reads the currently selected values of the ComboBox component.

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/mirajp/pen/YzzzKeX

#### Actual behavior:
Selected values are not read out loud.

#### Expected behavior:
When tabbing back to the combo box, the selected values should be read out loud (and the values should not appear to go blank when the field/control has focus).
![image](https://user-images.githubusercontent.com/6332917/66227525-2ca2ef00-e692-11e9-934e-6c6ceb19764e.png)

**Comment by khmakoto** [comment]
Hi @mirajp, thanks for filing this issue! I can confirm that I can repro this in your codepen above. I'll work on a fix as soon as I can. Thanks!

**Comment by khmakoto** [comment]
Hi @mirajp, I talked with @joschect about this behavior and we've decided to track it as part of the `ComboBox` and `Dropdown` refactor we've been thinking about. I've already made an entry there for this issue.

**Comment by micahgodbolt** [comment]
related to #10910
